### ISSUE TITLE
Why I get low profermance in practice (use web_cam)?

### ISSUE THREAD

**Issue by jario-jin** [foolwood/DaSiamRPN#59]
I modify the code 'demo.py' and use it for web_cam tracking application. The code I changed as follows:
```
# mouse callback function
def setting_bbox(event, x, y, flags, param):
    global lx, ly, cx, cy, w, h, start
    if event == cv2.EVENT_LBUTTONDOWN:
        lx, ly = x, y
        print('start = {}'.format(start))
    if event == cv2.EVENT_LBUTTONUP:
        cx = (x + lx)/2
        cy = (y + ly)/2
        w = x - lx + 1
        h = y - ly + 1
        start = True
        print('start = {}'.format(start))


cv2.namedWindow('SiamRPN')
cv2.setMouseCallback('SiamRPN', setting_bbox)
cap = cv2.VideoCapture(0)
```
I used both SiamRPNBIG and SiamRPNVOT,  I get very low profermance compared with KCF...
Is there anything wrong with my configuration?

**Comment by turboLIU** [comment]
I got low perfermance too~~~

**Comment by lwcode** [comment]
me too, except the 'bag' data in the project, it cannot track object very well,maybe something wrong with me?

**Comment by hkashyap** [comment]
I am having this issue too. When the target is occluded, then the tracker shifts to another object all the time. Sometimes, it shifts to other object without occlusion when the camera moves. I am using the demo code on the stream from an RGBD sensor (same model as demo VOT).
Any response regrading this performance will be highly appreciated.

**Comment by BaronH07** [comment]
> I am having this issue too. When the target is occluded, then the tracker shifts to another object all the time. Sometimes, it shifts to other object without occlusion when the camera moves. I am using the demo code on the stream from an RGBD sensor (same model as demo VOT).
> Any response regrading this performance will be highly appreciated.

In fact, instead of DaSiamRPN, I believe the codes stay in the SiamRPN which doesn't perform well in dealing with occlusion and identifying similar objects. It can be improved if you implement the real Distractor-aware stuff mentioned in their paper and trained the whole network again using their training techniques which are hard to reproduce without the help of authors.

**Comment by hkashyap** [comment]
@BaronH07 Yes, once I looked into the code, I saw that it is actually SiamRPN. So, distractor aware incremental learning and long-term strategies are not included. I hope the models given are trained using distractor (semantic negative) samples the paper writes.
